module/planning/GetUpPlanner/src/GetUpPlanner.cpp

```diff
@@ -49,6 +49,8 @@ namespace module::planning {
         on<Configuration>("GetUpPlanner.yaml").then([this](const Configuration& config) {
             this->log_level  = config["log_level"].as<NUClear::LogLevel>();
             cfg.fallen_angle = config["fallen_angle"].as<float>();
+            cfg.start_delay  = config["start_delay"].as<double>();
+            cfg._is_fallen   = config["_is_fallen"].as<bool>();
```

Comment:
```suggestion
            cfg._is_fallen   = config["is_fallen"].as<bool>();
```
why the `_`?

Reply:
also it looks like this key is nowhere to be found in your YAML file. YAML will probably throw a `Bad key` runtime error if you dont add this key in